docs/installation.md

```diff
@@ -1,52 +1,91 @@
 # Installation
 
 If you're looking for the latest stable version of the `sprocket` command line
-tool, you can download it from any of the [package managers](#package-managers)
-listed below. Otherwise, see the [build from source](#build-from-source) section
-on how to obtain and build a copy of the source code.
+tool, you can either [build it from source](#build-from-source) (most common),
+get Sprocket through a [package manager](#package-managers) (support still being
+added), or use [Docker](#docker).
 
-## Package Managers
+## Build From Source
 
-### Homebrew
+There are also a number of options to build `sprocket` from source, including
+pulling in the released source from [crates.io](#cratesio) or downloading the
+source directly from [GitHub](#github). 
 
-::: warning Notice
-While we'd like to make `sprocket` easily installable via [Homebrew], we're
-waiting to surpass the [75 star
-requirement](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff)
-for Homebrew formulas. If you feel so inclined, help us get there by starring [the
-repo](https://github.com/stjude-rust-labs/sprocket)!
-:::
+All methods for building `sprocket` from source require [Rust] and `cargo` to be
+installed. We recommend using [rustup] to accomplish this. 
 
 ### Crates.io
 
-Before you can build `sprocket`, you'll need to install [Rust]. We recommend
-using [rustup] to accomplish this. Once Rust is installed, you can install the
-latest version of `sprocket` by running the following command.
-
-::: code-group
+You can use `cargo` to install the latest version of `sprocket` from
+[crates.io].
 
 ```shell
 cargo install sprocket
 ```
 
-:::
-
-This will pull in the latest published version on [crates.io].
+If desired, you can also check out a specific version of `sprocket`.
 
+```shell
+cargo install sprocket@0.12.2
+```
 
-## Build From Source
+### GitHub
```

Comment:
Not mentioned here, but it's possible to just download the correct binary from the release page and run that directly. Also `cargo binstall` works (which is basically just a shortcut for the prior).

Reply:
I added details on downloading the released assets directly. I omitted `cargo-binstall` because I just don't think it's likely many people will have `cargo` preinstalled.